### PR TITLE
NOISSUE - Fix Docker for ARM

### PR DESCRIPTION
**Pull request by drasko**
Signed-off-by: drasko <drasko.draskovic@gmail.com>

### CODE REVIEW

File: scripts/ci.sh [hunk 1]
```diff
@@ -80,7 +80,7 @@ push() {
 		make -j$NPROC latest
 		docker system prune -a -f
 		install_qemu
-		make latest_arm
+		GOARCH=arm make -j$NPROC latest
```
Comment (blokovi) [inline scripts/ci.sh]: Why don't you set an appropriate GOARM value along with GOARCH?

File: Makefile [hunk 2]
```diff
@@ -7,23 +7,23 @@ BUILD_DIR = build
...
-	docker build --no-cache --build-arg SVC_NAME=$(subst docker_,,$(1)) --tag=mainflux/$(subst docker_,,$(1)) -f docker/Dockerfile .
-endef
-
-define make_docker_arm
-	docker build --no-cache --build-arg GOARCH=arm --build-arg GOARM=7 --build-arg SVC_NAME=$(subst docker_arm_,,$(1)) --tag=mainflux/$(subst docker_arm_,,$(1))-arm32v7 -f docker/Dockerfile .
+	docker build --no-cache \
+		--build-arg SVC=$(subst docker_,,$(1)) \
+		--build-arg GOARCH=$(GOARCH) \
+		--build-arg GOARM=$(GOARM) \
+		--tag=mainflux/$(subst docker_,,$(1))$(2) \
```
Comment (blokovi) [inline Makefile]: If names of arm images will have just suffix `-arm`, please rename them in docker/docker-compose.arm.yml

File: scripts/ci.sh [hunk 3]
```diff
@@ -70,17 +70,20 @@ install_qemu() {
 	MF_PATH=$GOPATH/src/github.com/mainflux/mainflux
 	cd $MF_PATH
 	sudo apt-get update && sudo apt-get -y install qemu-user-static
-	wget https://github.com/multiarch/qemu-user-static/releases/download/v2.11.1/qemu-arm-static.tar.gz  && tar -xzf qemu-arm-static.tar.gz && rm qemu-arm-static.tar.gz
-	sudo cp qemu-arm-static /usr/bin/
+	wget https://github.com/multiarch/qemu-user-static/releases/download/v2.11.1/qemu-arm-static.tar.gz  \
+		&& tar -xzf qemu-arm-static.tar.gz \
+		&& rm qemu-arm-static.tar.gz
+	sudo mv qemu-arm-static /usr/bin/
```
Comment (blokovi) [inline scripts/ci.sh]: `mqtt/Dockerfile.arm` demands this file in mainflux root, so you can't move it from here, at least not without other modifications

**Review by manuio — DISMISSED** [review]

**Review by manuio — DISMISSED** [review]
LGTM

File: scripts/ci.sh [hunk 4]
```diff
@@ -80,7 +82,9 @@ push() {
 		make -j$NPROC latest
 		docker system prune -a -f
 		install_qemu
-		make latest_arm
+		GOARCH=arm GOARM=7 make latest
```
Comment (drasko) [inline scripts/ci.sh]: Please paralelise the compilation.

Comment (blokovi) [reply scripts/ci.sh]: Tnx, it was removed in debuging process and then I put it back in wrong `make` command

**Review by dborovcanin — APPROVED** [review]

**Review by nmarcetic — APPROVED** [review]

**Pull request merged**